src/pages/msme/ViewCreditOptions.tsx

```diff
@@ -200,6 +204,23 @@ function ViewCreditOptions() {
                 options={SECTOR_TYPES}
                 placeholder={t('Sector')}
               />
+              <FormInput
+                className="w-3/5"
+                label={t('Annual Revenue')}
+                name="annual_revenue"
+                big={false}
+                type="currency"
+                placeholder={`${
+                  applicationContext.state.data?.borrower.currency ||
+                  ' ' ||
+                  applicationContext.state.data?.borrower.annual_revenue
```

Comment:
Other placeholders have a `t()` label, like

```
                  t('Award amount') || ' ' || applicationContext.state.data?.award.award_currency
```

Reply:
That makes sense for the award, as the field label is the amount requested but the default is the amount awarded. In this case, the label and the value refer to the same, so there is no need for the extra label in the placeholder.

Reply:
Cool. Now that I see the screenshot, I see that the placeholders are just about whatever is most helpful to the user, so is not expected to be consistent across fields.